lib/popolo_helper.rb

```diff
@@ -2,8 +2,12 @@
 
 module Popolo
   module Helper
-    def term_table_url(c, h, t)
-      '/%s/%s/term-table/%s.html' % [c[:slug].downcase, h[:slug].downcase, t[:csv][/term-(.*?).csv/, 1]]
+    def term_table_url(term)
+      '/%s/%s/term-table/%s.html' % [
+        term.country.slug.downcase,
+        term.legislature.slug.downcase,
+        term.csv_url[/term-(.*?).csv/, 1],
```

Comment:
Is there not a method on `term` that will get us a suitable ID here? If not there really should be…